astropy/cosmology/flrw/tests/test_lambdacdm.py

```diff
@@ -2,18 +2,11 @@
 
 """Testing :mod:`astropy.cosmology.flrw.lambdacdm`."""
 
-##############################################################################
-# IMPORTS
-
-# STDLIB
-
-# THIRD PARTY
 import pathlib
```

Comment:
We are using Ruff to keep the imports sorted, and Ruff doesn't know how to sort comments (why should it?), so we can easily end up with cases like this where the comment is actively lying instead of being helpful. It would be good to get rid of these comments in all of `astropy`.